app/views/users/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Here is a list for all saved users</h1>
\ No newline at end of file
+<div class='user-index'>
+  <h1>Users</h1>
+
+  <% @users.each do |user| %>
+    <a class='colored-a' href="/users/<%= user.id %>">
+      <div class='user-div'>
+        <div>
+          <img src="<%= user.photo %>" />
+        </div>
+        <div class='name-and-post'>
+          <p><%= user.name %><p>
+          <p>Number of posts: <%= user.posts.count %></p>
+        </div>
+      </div>
+    </a>
+  <% end %>
+</div>
```

Comment:
## [OPTIONAL]

- [ ] Nice work displaying users' information here but for best practice, I would suggest you the option below  instead of using `@users.each`
```html.erb
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```